imod/mf6/wel.py

```diff
@@ -248,7 +320,7 @@ def _create_cellid(
         df_for_cellid = assigned_wells.groupby("index").first()
         d_for_cellid = df_for_cellid[["x", "y", "layer"]].to_dict("list")
 
-        return self._derive_cellid_from_points(like, **d_for_cellid)
+        return derive_cellid_from_points(like, **d_for_cellid)
```

Comment:
If the well is ever moved the cellid needs to be updated. 
Thus method then needs to be called whenever the row,column or layer is adjusted

I don't know how likely that is but i cant imagine a use cage where you start with a base model with wells and then move some of them to see the effect of it.